chia/wallet/puzzles/cat.clvm

```diff
@@ -242,43 +246,40 @@
 
 
       ;; This next section of code loops through all of the conditions to do three things:
-      ;;   1) Look for a "magic" value of -113 and, if one exists, take note of the tail reveal and solution
+      ;;   1) Look for a "magic" value of -113 and, if one exists, filter it, and take note of the tail reveal and solution
       ;;   2) Morph any CREATE_COIN or CREATE_COIN_ANNOUNCEMENT conditions
       ;;   3) Sum the total output amount of all of the CREATE_COINs that are output by the inner puzzle
       ;;
       ;; After everything return a struct in the format (morphed_conditions . (output_sum . tail_reveal_and_solution))
-      
+      ;; If multiple magic conditions are specified, the later one will take precedence
```

Comment:
Isn't it the earlier one because we're coming back up the recursive stack?
It's evaluated later, but exists earlier in the list

Reply:
No, we take note of the info when we are heading down and then we carry the last value we found up with us through the recursive stack.